core/src/consensus/stake/mod.rs

```diff
@@ -208,6 +221,19 @@ fn delegate_ccs(
     Ok(())
 }
 
+fn revoke(state: &mut TopLevelState, sender: &Address, delegatee: &Address, quantity: u64) -> StateResult<()> {
+    let mut delegator = StakeAccount::load_from_state(state, sender)?;
+    let mut delegation = Delegation::load_from_state(state, &sender)?;
+
+    delegator.add_balance(quantity)?;
+    delegation.subtract_quantity(*delegatee, quantity)?;
+    // delegation does not touch stakeholders
```

Comment:
Do we need this comment?

Reply:
It would be an answer to a question 'It touches the balance while delegate/revoke delegation. Should I update stakeholder list?'.